torch/csrc/api/include/torch/nn/utils/clip_grad.h

```diff
@@ -13,7 +13,8 @@ namespace utils {
 inline double clip_grad_norm_(
```

Comment:
I'm curious about why the C++ and python APIs have different impls. Maybe the python one should be removed eventually